addons/main/script_component.hpp

```diff
@@ -0,0 +1,16 @@
+#define COMPONENT main
+#include "\z\diwako_dui\addons\main\script_mod.hpp"
+
+// #define DEBUG_MODE_FULL
+// #define DISABLE_COMPILE_CACHE
+// #define CBA_DEBUG_SYNCHRONOUS
+// #define ENABLE_PERFORMANCE_COUNTERS
```

Comment:
No need to put macros here that you don't use. `CBA_DEBUG_SYNCHRONOUS` has 0 occurences in ACE and CBA, where did you get that from?

Reply:
`CBA_DEBUG_SYNCHRONOUS` came from the acemod arma-project-template repository. Which is what HEMTT uses for it's base files